proxy/proxy/broker/rhnRepository.py

```diff
@@ -462,8 +473,8 @@ def cache(stringObject, directory, filename, version):
         # Try to create this new file
         try:
             fd = os.open(tempfile, os.O_WRONLY | os.O_CREAT | os.O_EXCL,
-                         0644)
-        except OSError, e:
+                         0o644)
```

Comment:
Since you're touching this already, please put back this silly newline to the previous statement.